documentation/Readme.md

```diff
@@ -0,0 +1,47 @@
+# Project Title
+- Cool logo
+
+---
+
+### Introduction
+
+- Summary of the purpose and function of this project
+- Provide links to Issue queue and project page (if any)
+
+### Requirements
+
+- Hardware requirements (Processor/Storage/RAM/GPU)
+- OS requirements (List of compatible Operating Systems)
+- List of Dependencies
+
+### Installation
+
+- Guide/code to install/build dependencies for respective OS
+- If a dependency is to be built, provide a link to its *Installation* page. Do not reinvent the wheel here.
+- Instructions to build the project after the dependencies have been met
+
+### Configuration
+
+- List any configuration is required; for example, the various config options in cmake.
+- Any special notes about the configuration of this module – including but not limited to permissions.
+
+### Testing
+
+- Instructions to run unit tests.
+- Each unit test should have internal documentation on what the test is checking for, and the logic that is being used.
+
+- Instructions for Integration Testing
+
+- Instructions to follow when tests fail. (This may also be included int eh Troubleshooting section)
+
+### Troubleshooting and FAQ
+This sections builds up over time.
+
+- List out common problems that users face, along with solutions.
+- What to do when something breaks.
+
+### Maintainers
+
+- Names of people associated with the project split into two lists :
+  - Active
```

Comment:
What order should the names be in, Alphabetical order or on the basis of the number of contributions?

Reply:
I'd prefer alphabetical order. Added it. What do you feel?

Reply:
What do you think is the purpose of writing a contributor name in README?

Reply:
It can be a way of : 
- Giving credit where it is due
- Showing gratitude to collaborators
- Making someone feel responsible
- Showing whom to contact in case someone needs help

Reply:
If it is for "Showing whom to contact in case someone needs help". Then I think that we need to put it in such an order in which the person who can help most (someone who has time and complete knowledge of the project) should be at the top of the list.
One other way is to just keep it in alphabetical order and for each developer, we should mention what sort of help he can give or what sort of problems he can resolve.